drizzlepac/hlautils/product.py

```diff
@@ -23,6 +24,14 @@
 log = logutil.create_logger(__name__, level=logutil.logging.NOTSET, stream=sys.stdout,
                             format=SPLUNK_MSG_FORMAT, datefmt=MSG_DATEFMT)
 
+# Define keywords to be added to SVM products to describe the overlap of
+# exposures onto the output WCS footprint as determined from the SkyFootprint masks
+MASK_KWS = {"NPIXFRAC": [None, "Fraction of pixels with data"],
+            "MEANEXPT": [None, "Mean exposure time per pixel with data"],
+            "MEDEXPT": [None, "Median exposure time per pixel with data"],
+            "MEANNEXP": [None, "Mean exposures per pixel with data"],
+            "MEDNEXP": [None, "Median exposures per pixel with data"],
```

Comment:
Clarify the commentary part of the keyword for MEANNEXP and MEDNEXP - "[Mean|Median] **_number of_** exposures..."

Reply:
Fixed in b421222.